modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/api/lifecycle/APIVisibilityByDomainTestCase.java

```diff
@@ -78,103 +79,111 @@ public void initialize() throws Exception {
         apiEndPointUrl = backEndServerUrl.getWebAppURLHttp() + API_END_POINT_POSTFIX_URL;
         storeURLHttp = getStoreURLHttp();
         //Login to API Publisher and Store with CarbonSuper admin
-        apiPublisherClientCarbonSuperAdmin = new APIPublisherRestClient(getPublisherURLHttp());
-        apiStoreClientCarbonSuperAdmin = new APIStoreRestClient(storeURLHttp);
-        apiPublisherClientCarbonSuperAdmin.login(user.getUserName(),user.getPassword());
-        apiStoreClientCarbonSuperAdmin.login(user.getUserName(), user.getPassword());
+        restAPIPublisherCarbonSuperAdmin = new RestAPIPublisherImpl("admin", "admin",
+                SUPER_TENANT_DOMAIN_NAME, keyManagerHTTPSURL, gatewayHTTPSURL, publisherURLHttp);
```

Comment:
Lets use it as MultitenantConstants.SUPER_TENANT_DOMAIN_NAME

Reply:
Fixed